plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -87,6 +87,8 @@ func defaultPriorities() sets.String {
 	return sets.NewString(
 		// Prioritize nodes by least requested utilization.
 		factory.RegisterPriorityFunction("LeastRequestedPriority", priorities.LeastRequestedPriority, 1),
+		// Prioritize nodes by image locality.
```

Comment:
I think we should not make this part of default, because I am worried about performance impact.

Once we have priority score caching (#17390), the performance impact will be small because you will only update priority when something changes on a node. But right now, every time scheduler looks at a pending pod, it will have to look at images on every node, which I think is probably too much work. (You are welcome to run density and scalability e2es with and without your change to see what is the performance impact, but I would suggest we merge your PR with not-default policy, and then make it default later if performance impact is determined to be OK.)